<launch>
        <param name="robot_description" command="cat $(find cepheus_robot)/model/cepheus.urdf" />   

        <rosparam command="load" file="$(find cepheus_robot)/config/controllers.yaml"/>

        <!--node name="encoder_card_drivers_loader" pkg="cepheus_robot" type="DM35820_driver_load.sh" output="screen">
        </node-->
        
        <node name="mouse_odom_publisher" pkg="cepheus_robot" type="mouse_odom_node" output="screen">
                <param name="left_dev_event" value="/dev/input/by-path/pci-0000:00:1d.7-usb-0:1.3:1.0-event-mouse"/>
                <param name="right_dev_event" value="/dev/input/by-path/pci-0000:00:1d.7-usb-0:1.4:1.0-event-mouse"/>
        </node>

        <!--node name="phase_space_inverted_node" pkg="cepheus_robot" type="phase_space_inverted_node" output="screen">
        </node-->

        <node name="camera_odom_publisher" pkg="cepheus_robot" type="camera_odom_node" output="screen">
                <param name="robot_id" value="11" />
                <param name="udp_port" value="25000" />
        </node>

        <!--node name="thrusters_node" pkg="cepheus_robot" type="thrusters_node" output="screen">
                <param name="frequency" value="10" />
                <param name="resolution" value="100" />
        </node-->

        <node name="cepheus_interface_node" pkg="cepheus_robot" type="cepheus_interface_node" respawn="true" output="screen">
        </node>

        <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="joint_state_publisher reaction_wheel_velocity_controller" >
        </node>

        <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher">
                <!--param name="tf_prefix" value="servicer" /-->
                <!--param name="publish_frequency" value="50" -->
        </node>  


        <!--include file="$(find phase_space)/launch/track_object.launch">
        </include-->

</launch>
